pkg/util/utils.go

```diff
@@ -682,18 +682,21 @@ func DefaultContainerConfig() *config.Config {
 	return containerConfig
 }
 
-func CreateCidFile(cidfile string, id string) error {
-	cidFile, err := OpenExclusiveFile(cidfile)
-	if err != nil {
-		if os.IsExist(err) {
-			return fmt.Errorf("container id file exists. Ensure another container is not using it or delete %s", cidfile)
+func CreateIDFile(path string, id string) error {
+	baseDir := filepath.Dir(path)
+	if baseDir != "" {
+		if _, err := os.Stat(baseDir); err != nil {
+			return err
 		}
-		return fmt.Errorf("opening cidfile %s", cidfile)
 	}
+	cidFile, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0666)
+	if err != nil {
+		return fmt.Errorf("opening cidfile %s", path)
```

Comment:
Why does this not return the actual error? This makes it impossible to know what exactly caused it to fail.

Reply:
Would you also consider changing `opening` to `creating`? Even though I'm old enough to know better, my brain can't help but interpret `opening` as `...opening for read`.